quantized_mesh_tile/editable_terrain.py

```diff
@@ -0,0 +1,476 @@
+# -*- coding: utf-8 -*-
+import numpy
+import os
+
+import numpy as np
+from future.utils import old_div
+
+from quantized_mesh_tile import TerrainTile
+from quantized_mesh_tile.llh_ecef import LLH2ECEF
+from quantized_mesh_tile.terrain import lerp
+from quantized_mesh_tile.utils import triangleArea
+from . import cartesian3d as c3d
+
+null_normal = [0, 0, 0]
+np_null_normal = np.array([0, 0, 0])
+
+
+class EditableTerrainTile(TerrainTile):
+    """
+    Helper class to edit a given terrain tile on the edges.
+    Changes are possible on specified Heights, Triangles and Normals.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(EditableTerrainTile, self).__init__(*args, **kwargs)
+        self.isIndexDirty = False
+        self._changedHeights = []
+        self._filePath = None
+        self._gzipped = False
+        self.name = None
+
+    def setName(self, name):
+        self.name = name
+
+    def getEdgeIndices(self, edge):
+        """
+        Returns the indices of vertices on the defined edge.
+        :param edge: the edge of the tile. Specified by the flags 'w','n','e','s'
+                    for west-, north-, east- and  south-edge
+        :return: Array of integers
+        """
+        if 'w' == edge:
```

Comment:
you might want to support both `w` and `west` since your return these keys in `boundingBox`.

maybe `edge in ('w', 'west')`

Reply:
That would be the better solution my initial thought was to ensure that only one kind of specification should be applied in the usages (either only 'w' or only 'west'), but your solution is less error-prone, i think.